src/test/java/io/confluent/connect/jdbc/source/integration/PostgresOOMIT.java

```diff
@@ -24,87 +22,46 @@
 import org.slf4j.LoggerFactory;
 
 /**
- * Integration test for Postgres OOM conditions
- * Expects to be run with -Xmx64M or -Xmx128M for minimum runtime
- * but will behave correctly with any heap size (just with less performance)
+ * Integration test for Postgres OOM conditions Expects to be run with -Xmx64M or -Xmx128M for
+ * minimum runtime but will behave correctly with any heap size (just with less performance)
```

Comment:
I think you can promote the second half of this comment up to the base OOMIT class